src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.Node.cs

```diff
@@ -50,9 +49,9 @@ private string GetDebuggerDisplay()
         private struct Node
         {
```

Comment:
`readonly struct` as long as we're here

Reply:
I've been leaving this for [EPS01](https://github.com/SergeyTeplyakov/ErrorProne.NET/blob/775522226c2d17018e4379c2280d18ddfca06888/src/ErrorProne.NET.StructAnalyzers/MakeStructReadOnlyAnalyzer.cs#L8-L12), either by referencing that analyzer package or porting the rule to dotnet/roslyn-analyzers.